mmdet/apis/train.py

```diff
@@ -196,6 +196,12 @@ def train_detector(model,
         runner.register_hook(
             eval_hook(val_dataloader, **eval_cfg), priority='LOW')
 
+    resume_from = None
+    if cfg.resume_from is None and cfg.auto_resume:
```

Comment:
Do not use `cfg.auto_resume`, it will cause BC breaking. Use cfg.get('auto_resume') instead.

Reply:
Done.